pilot/pkg/networking/core/v1alpha3/route/route.go

```diff
@@ -192,7 +192,7 @@ func buildVirtualHostsForVirtualService(
 	}
 	out := make([]VirtualHostWrapper, 0, len(serviceByPort))
 	for port, portServices := range serviceByPort {
-		routes, err := BuildHTTPRoutesForVirtualService(virtualService, serviceRegistry, port, proxyLabels, gatewayName)
+		routes, err := BuildHTTPRoutesForVirtualService(virtualService, serviceRegistry, port, proxyLabels, gatewayName, nil)
```

Comment:
you aren't passing the config store here.

Reply:
nil here as this specific call is for sidecar - no ring hash available there we assume.

Reply:
We aren't passing in the config store here because this is only used for building sidecar outbound http route configs. We could add different function signatures to make this clearer.